unifi/device.go

```diff
@@ -2,6 +2,22 @@ package unifi
 
 import (
 	"context"
+	"fmt"
+)
+
+const (
```

Comment:
I don't need all of these so if you'd prefer I can get rid of the unused constants.

Reply:
If they would be useful for other folks using the SDK I think its fine. I don't know if we have this precedent anywhere, but I think the const's [should still be `MixedCaps`](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps), ie. `DeviceStateUnknown`? But I don't really feel that strongly, so if you do, ok with me.

Reply:
You could also create a type for this to make the usage more obvious, ie:
```go
type DeviceState int

const (
  DeviceStateUnknown DeviceState = 0
  DeviceStateConnected
  DeviceStatePending
  // ...
)

type Device struct {
  // ...
  State DeviceState
  // ...
}
```